go/stats/counters.go

```diff
@@ -205,23 +221,32 @@ func (mc *MultiCounters) Set(names []string, value int64) {
 type MultiCountersFunc struct {
 	CountersFunc
 	labels []string
+	help   string
 }
 
 // Labels returns the list of labels.
 func (mcf *MultiCountersFunc) Labels() []string {
 	return mcf.labels
 }
 
+func (mcf *MultiCountersFunc) Help() string {
```

Comment:
I think you don't need this if the Help() method is defined on the Counters struct.
It also seems like we want a corresponding Name() function at the same level.

Reply:
I don't think we need Name() (or to store it), so I've gotten rid of it.